### PR TITLE
[ADD] Draft Security Policy

### PR DESCRIPTION
**Pull request by odony**
Short version mostly linking to the full responsible disclosure policy.

### CODE REVIEW

**Comment by Yenthe666** [comment]
🎈 🎉

**Comment by sswapnesh** [comment]
:heart:

**Comment by sswapnesh** [comment]
Are we also going to Publish security vulnerabilities on https://github.com/odoo/odoo/security/advisories ?

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+## Important note
+
+We receive a majority of security reports that have little to no impact on the security of Odoo or
+the Odoo Cloud, and we ultimately have to reject them. To avoid a disappointing experience when
+contacting us, please try to put together a proof-of-concept attack and take a critical look at
+what's really at risk.
+If the proposed attack scenario turns out unrealistic, your report will probably be rejected.
+Also be sure to review our list of [non-qualifying issues](https://www.odoo.com/security-report#what).
```
Comment (sswapnesh) [inline SECURITY.md]: or better to redirect at https://www.odoo.com/security-report#nonqualifying
```suggestion
Also be sure to review our list of [non-qualifying issues](https://www.odoo.com/security-report#nonqualifying).


```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 12.0    | :white_check_mark: |
```
Comment (sswapnesh) [inline SECURITY.md]:
```suggestion
| 12.0    | :heavy_check_mark: |
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+| 11.0    | :white_check_mark: |
+| 10.0    | :white_check_mark: |
+| <10.0   | :x:                |
```
Comment (fmdl) [inline SECURITY.md]: <=9.0, it is more readable

**Pull request merged**